src/eduid_webapp/signup/views.py

```diff
@@ -52,19 +53,17 @@
 @signup_views.route('/config', methods=['GET'])
 @MarshalWith(FluxStandardAction)
 def get_config():
-    url = current_app.config.get('INTERNAL_ACTIONS_URL')
+    tou_url = urlappend(current_app.config.get('TOU_URL'), 'get-tous')
     try:
-        r = http.request('GET', url + 'get-tous', retries=False)
-        current_app.logger.debug('Response: {!r} with headers: '
-                '{!r}'.format(r, r.headers))
+        r = http.request('GET', tou_url, retries=False)
+        current_app.logger.debug('Response: {!r} with headers: {!r}'.format(r, r.headers))
+        # XXX: Is the 302 check really needed? Shouldn't TOUs be public?
```

Comment:
ToUs are public. But if the actions app does not see a session cookie in the request it will redirect to itself with a set-cookie header, so here we are resending the request adding the cookie.